governance/polls/Initial Parameters for StarkNet Simple Bridge - March 28, 2022.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Initial Parameters for StarkNet Simple Bridge - March 28, 2022
+summary: Signal your support or opposition to the following initial parameters, designed for the deployment of StarkNet Simple Bridge.
+discussion_link: https://forum.makerdao.com/t/starknet-simple-bridge-limit-poll-risk-assessment-mitigation/14158
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-03-28T16:00:00
+end_date: 2022-03-31T16:00:00
+---
+# Poll:Initial Parameters for StarkNet Simple Bridge - March 28, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the MakerDAO Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, March 28 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes: 
+* Implementing the following parameters for the deployment of StarkNet Simple Bridge:
+
+**Proposed Initial Parameters**
```

Comment:
Not sure all of these are parameters. Maybe take some inspiration from here: https://github.com/makerdao/community/blob/master/governance/polls/Optimism%20DAI%20Bridge%20-%20March%2015%2C%202021.md. That said, there are definitely improvements to be made compared to how we did it with Optimism.

Reply:
I'd probably just lose this heading and keep the bullets.